vb_suite/reshape.py

```diff
@@ -55,7 +55,7 @@ def unpivot(frame):
 # Melt
 
 setup = common_setup + """
-from pandas.core.reshape import melt
```

Comment:
I know it was an automatic search replace, but no need to change those (we should actually just remove them)